doc/DeveloperGuide.md

```diff
@@ -67,6 +67,23 @@ Priority | As a ... | I want to ... | So that I can...
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+System: Addressbook-level3
+Use case: UC01 - Rename Tag
+Actor: User
+MSS:
+    1. user requires to rename a tag
+    2. System ask user to confirm rename
+    3. user confirm to rename the tag
+    4. System rename the tag and updates data
+    5. System shows successful message
```

Comment:
Use cases should be interactions. Hence point 5 cannot be System as it does not show interaction between User and System.